lang/LangSource/PyrSlot64.h

```diff
@@ -50,7 +52,7 @@ enum {
 };
 
 typedef struct pyrslot {
-	long tag;
+	int64 tag;
```

Comment:
possibly. afaict this is compiler dependent: the compiler could decide to add some padding between tag and union to ensure 64bit alignment of the union. iirc mingw and msvc used to have different rules for padding structs and mingw can emulate the msvc behavior with -ms-bitfields.

using a signed integer here, we ensure that `sizeof(PyrSlot)` is 128bit and we can use sse instructions to copy slots